frontend/src/lib/logic/featureFlagLogic.ts

```diff
@@ -10,6 +10,45 @@ import posthog from 'posthog-js'
 
 type FeatureFlagsSet = { [flag: string]: boolean }
 
+const eventsNotified: Record<string, boolean> = {}
+function notifyFlagIfNeeded(flag: string, flagState: boolean): void {
```

Comment:
Why aren't we just using the native `posthog.isFeatureEnabled` instead of featureFlagLogic?

Reply:
If we put this inside a react render component, then in my tests, the action and navigation feature flags were checked 6 times per render and hundreds of times when browsing around. That's too many events.